src/Metadata/Resource/ResourceMetadata.php

```diff
@@ -41,6 +42,7 @@ public function __construct(string $shortName = null, string $description = null
         $this->subresourceOperations = $subresourceOperations;
         $this->graphql = $graphql;
         $this->attributes = $attributes;
+        $this->sunset = $sunset;
```

Comment:
This is not necessary IMO, an attribute would be better suited.

Reply:
removed